modules/howtos/examples/collection-manager.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+use \Couchbase\ClusterOptions;
+use \Couchbase\Cluster;
+use \Couchbase\User;
+
+function getCollections($username, $password) {
+    print "create-collection-manager\n";
+
+	// tag::create-collection-manager[]
+    $options = new \Couchbase\ClusterOptions();
+    $options->credentials($username, $password);
+    $cluster = new \Couchbase\Cluster("localhost", $options);
+    $bucket = $cluster->bucket("travel-sample");
+
+    $collections = $bucket->collections();
+	// end::create-collection-manager[]
+
+	return $collections;
+}
+
+function main() {
+    $connectionString = "couchbase://localhost";
+    $options = new \Couchbase\ClusterOptions();
+    $options->credentials("Administrator", "password");
+    $cluster = new \Couchbase\Cluster($connectionString, $options);
+
+    $bucket = $cluster->bucket("travel-sample");
+
+    $users = $cluster->users();
+
+    print "bucketAdmin\n";
+    // tag::bucketAdmin[]
+    $user = new \Couchbase\User();
+    $user->setUsername("bucketAdmin");
+    $user->setDisplayName("Bucket Admin [travel-sample]");
+    $user->setPassword("password");
+    $user->setRoles([
+        (new \Couchbase\Role)->setName("bucket_admin")->setBucket("travel-sample")
+    ]);
+
+    $users->upsertUser($user);
+    // end::bucketAdmin[]
+
+    $collections = getCollections("bucketAdmin", "password");
+
+    print "create-scope\n";
+    // tag::create-scope[]
+    try {
+        $collections->createScope("example-scope");
+    }
+    catch (Exception $e) {
+        print $e;
+    }
+    // end::create-scope[]
+
+    print "scopeAdmin\n";
+	// tag::scopeAdmin[]
```

Comment:
got an extra space here I think?

Reply:
woo, so what's happened is that Go uses tabs, and I copied the examples from the Go project while translating them. So any thing I typed new was with spaces, but the tags were left unchanged, so stayed as tabs...

Didn't show up on my editor of course... will fix now, thanks.